src/components/menu/common/menu-action-common.ts

```diff
@@ -27,7 +27,7 @@ export const SbbMenuActionCommonElementMixin = <
   superClass: T,
 ): AbstractConstructor<SbbMenuActionCommonElementMixinType> & T => {
   abstract class SbbMenuActionCommonElement
-    extends SbbIconNameMixin(SbbDisabledTabIndexActionMixin(superClass))
+    extends SbbIconNameMixin(SbbDisabledMixin(superClass))
```

Comment:
I think this is correct for menu-link, but what about menu-button?

Reply:
Good catch, you're correct